[docs] FAQ update (React Native) (#5728)
DevExpress/testcafe: docs/articles/faq/README.md

See the [Select Elements With Dynamic IDs](../documentation/guides/basic-guides/select-page-elements.md#select-elements-with-dynamic-ids) example for details.

### Can I use TestCafe to test React Native apps?

> No. You can use TestCafe to test anything that runs in a browser — including Progressive Web Applications and Electron apps. TestCafe can not automate native mobile applications, because they do not run in a browser.

Reviewer: ```suggestion
You can only use TestCafe to test something that runs in a browser, including Progressive Web Applications and Electron apps. TestCafe cannot automate native mobile applications because they do not run in a browser.
```
